src/clients/nfs-client4part.rb

```diff
@@ -62,8 +62,6 @@ def main
 
         if Builtins.contains(@w_ids, Convert.to_symbol(@widget_id))
           return ToStorage(@modify_line)
-        else
-          return nil
```

Comment:
see https://github.com/yast/yast-nfs-client/pull/51/files/3dccf898d3c2490de1cbae4d9588db32d2dee760#diff-9c47915e3147e857bb6467c6946cb7a6L70

it already returning nil explicitelly